src/store/groups/reducers.ts

```diff
@@ -20,15 +21,17 @@ export function groupReducer(
           tableView: {
             selectedGroupId: action.payload,
             table: []
-          }
+          },
+          fetching: true
         }
       case FETCH_GROUP_TABLE_SUCCESS:
```

Comment:
please add also FETCH_GROUP_TABLE_FAILURE case to set `fetching: false` in case of an error

Reply:
ok, I was planning on opening a separate PR for this :)